gc_ruboconfig.gemspec

```
@@ -2,15 +2,15 @@
 
 Gem::Specification.new do |spec|
   spec.name          = 'gc_ruboconfig'
-  spec.version       = '2.14.0'
+  spec.version       = '2.15.0'
   spec.summary       = "GoCardless's shared Rubocop configuration, conforming to our house style"
   spec.authors       = %w[GoCardless]
   spec.homepage      = 'https://github.com/gocardless/ruboconfig'
   spec.email         = %w[developers@gocardless.com]
   spec.license       = 'MIT'
 
   spec.files         = 'rubocop.yml'
-  spec.add_dependency 'rubocop', '>= 0.84'
+  spec.add_dependency 'rubocop', '>= 0.85'
```

Being consistent with our required versions elsewhere:

```suggestion
  spec.add_dependency 'rubocop', '>= 0.87'
```